src/index.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!Doctype html>
```

Comment:
The `<!DOCTYPE html>` declaration should be in uppercase (`<!DOCTYPE html>`) to ensure compatibility across different browsers.